labelbox/schema/project.py

```diff
@@ -78,9 +78,25 @@ class Project(DbObject, Updateable, Deletable):
     predictions = Relationship.ToMany("Prediction", False)
     ontology = Relationship.ToOne("Ontology", True)
 
+    def members(self):
+        """ Fetch all current members for this project
+
+        Returns:
+            A `PaginatedCollection of `ProjectMember`s
+
+        """
+        id_param = "projectId"
+        query_str = """query ProjectMemberOverviewPyApi($%s: ID!) {
+             project(where: {id : $%s}) { id members(skip: %%d first: %%d){ id user { %s } role { id name } }
+           }
+        }""" % (id_param, id_param, query.results_query_part(Entity.User))
+        return PaginatedCollection(
+            self.client, query_str, {id_param: str(self.uid)},
+            ["project", "members"],
+            lambda client, res: ProjectMember(client, res))
```

Comment:
Why does `PaginatedCollection` accept a lambda? Don't see a `Callable` in `PaginatedCollection`'s __init__

Reply:
It accepts a DBObject which is a callable that accepts a client and json response. This pattern has been used elsewhere in the SDK to customize the parsing of the response string. However, it isn't necessary here anymore. I'll update this to just accept `ProjectMember`.

Reply:
Fixed.